Program.cs

```diff
@@ -1,25 +1,41 @@
 ﻿const string filePath = "input.txt";
-
+int letter_count = 0;
+int DigitCount = 0;
 try
 {
-    if (File.Exists(filePath))
+    using var reader = new StreamReader(filePath);
+
+    while (!reader.EndOfStream)
     {
-        var content = File.ReadAllText(filePath);
 
-        int letterCount = content.Count(char.IsLetter);
-        int digitCount = content.Count(char.IsDigit);   
+        var line = reader.ReadLine()!;
 
-        Console.WriteLine($"The file contains {letterCount} letters and {digitCount} digits.");
-    }
-    else
-    {
-        Console.WriteLine($"File not found at path: {filePath}");
-    }
+        foreach (var character in line)
+        {
+            var helper = new Helper();
+
+            var isCharacter = helper.IsCharacterAndIfINeedToCount(character);
+            var isDigit = helper.IsDigitAndIfItIsInTheList(character);
+
+            IncreaseLetter_CountIfTrue(letter_count, isCharacter);
+            helper.IncreaseDigitCount(DigitCount, isDigit);
+        }
+        
+        void IncreaseLetter_CountIfTrue(int letter_countt, bool isCharacter)
+        {
+            letter_count = isCharacter ? ++letter_count : letter_count;
+        }
+    }       
 }
-catch (Exception ex)
+catch (FileNotFoundException)
 {
-    Console.WriteLine($"An error occurred: {ex.Message}");
+    Console.WriteLine($"File was not found in provided path");
 }
 
+Console.WriteLine($"The file contains {letter_count} letters and {DigitCount} digits.");
+Console.ReadKey();
```

Comment:
Readkey stops application and user input is not used

Reply:
That's why it was used, to cover this:
`The console should print first how many digits and characters where in provided input and after enter was pressed how many symbols are in total`

Reply:
you have console write, console write, console read (not used), console write, console read (to stop program from closing and user to see outcome, that why you do not need Readkey at line 36

Reply:
`The requirement is to print first line and after enter is pressed print second line.`
Console.ReadKey obtains the next character or function key pressed by the user. So why do not I need to readkey on 36line.?

Reply:
Then it would be better to indicate user what program expects with console write "Press enter to see symbol count" or something similar